metadata/src/main/java/com/redhat/lightblue/metadata/ValueGenerator.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.lightblue.metadata;
+
+import java.io.Serializable;
+import java.util.Properties;
+
+/**
+ * Represents value generator definition located in entityInfo.
+ *
+ * @author mpatercz
+ *
+ */
+public class ValueGenerator implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    public enum ValueGeneratorType {
+        IntSequance;
+    }
+
```

Comment:
SimpleField already has a type defined. Isn't that enough? Will it help with the implementation on the lightblue-mongo side to have com.redhat.lightblue.metadata.Type set on the generator itself?

I'm not sure if using Sequence instead of IntSequence is a good idea. Generation type heavily depends on the kind of the value being generated. Value type is not just a generation parameter, e.g. a random string will have a different implementation than random decimal. ValueGeneratorTypes like RandomDecimal, RandomString make sense to me, at least you can easily see what generators are available, rather than try out different combinations of generation type and value type.